### ISSUE TITLE
Ability to copy/share QR code image

### ISSUE THREAD

**Issue by yanascz** [ZeusLN/zeus#1285]
### Describe the feature

As a user, I'd like to be able to copy/share a QR code image. E.g. in Phoenix, I can long-tap a QR code image and an action pop-up appears to copy/share/enlarge the image.

Essentially, being able to copy the image would be awesome. Sharing option is just nice to have.

**Comment by Smiril** [comment]
Hello would a screenshot do it too for sharing???

and crop them on your phone …. On iOS do screenshot by hold „Home Button an powerbutton“ to crop tap on the popup image left down side

**Comment by yanascz** [comment]
> Hello would a screenshot do it too for sharing???

Yep, that's of course doable but it's not great from UX perspective. 😞

**Comment by Smiril** [comment]
But working for now

**Comment by yanascz** [comment]
> But working for now

That's right. 👍 However, it would be awesome to have a more UX friendly option eventually.
